webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/services/CasProductJaxrsService.java

```diff
@@ -46,88 +33,99 @@
 import org.apache.oodt.cas.product.jaxrs.resources.TransferResource;
 import org.apache.oodt.cas.product.jaxrs.resources.TransfersResource;
 
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Vector;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.servlet.ServletContext;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+
 /**
- * Service class that handles HTTP requests and returns file manager entities such as {@link
- * Reference references} and {@link Product products} as JAX-RS resources converted to different
- * formats.
- *
+ * Service class that handles HTTP requests and returns file manager entities
+ * such as {@link Reference references} and {@link Product products} as
+ * JAX-RS resources converted to different formats.
  * @author rlaidlaw
  * @version $Revision$
  */
-public class CasProductJaxrsService {
-  private static final Logger LOGGER = Logger.getLogger(CasProductJaxrsService.class.getName());
+public class CasProductJaxrsService
```

Comment:
Can you reformat this class with google code formatter? Seems like most of these changes are formatting changes. I will check this once you update this class to include only the required changes.